guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -94,7 +94,15 @@ Reloading the page only triggers compilation and publication of the styles used
 
 ##### Debugging using the static view files deployment tool
 
-Once you save your changes and run the `magento setup:static-content:deploy` cli command from the `<your_Magento_instance>/bin` directory, the tool pre-processes (including compilation) and publishes the static view files.
+Once you save your changes and run the next cli command in the root of your `<your_Magento_instance>` directory:
+```text
+bin/magento setup:static-content:deploy
+``` 
+
+The tool pre-processes (including compilation) and publishes the static view files.
```

Comment:
I believe we continue the sentence here. So, no need to capitalize the first word

Reply:
**fixed**

Thank you!